gui/widgets/editable.cpp

```diff
@@ -129,49 +138,86 @@ bool EditableWidget::handleKeyDown(Common::KeyState state) {
 	switch (state.keycode) {
 	case Common::KEYCODE_RETURN:
 	case Common::KEYCODE_KP_ENTER:
+		initHighlight();
```

Comment:
the name of the method is misleading. "init" normally means start. So, better name it `clearHighLight()`